x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -94,14 +112,18 @@ export function ServiceOverviewTransactionsTable({ serviceName }: Props) {
       end,
       transactionType,
       latencyAggregationType,
+      comparisonType,
+      pageIndex,
+      direction,
+      field,
```

Comment:
@dgieselaar I can calculate the visible transaction groups inside useFetcher, but I'll still need the `react-hooks/exhaustive-deps`, because `comparisonType` needs to be listed on its dependencies, even though it is not used, to trigger the comparison API call when it's changed.